packages/frontend/src/theme.tsx

```diff
@@ -0,0 +1,14 @@
+import { createTheme } from '@nextui-org/react';
+
+export const mainTheme = createTheme({
+  type: 'light',
+  theme: {
+    colors: {
+      primaryLight: '#99f6e4', //teal200
+      primary: '#2dd4bf', //teal400
+      primaryDark: '#0d9488', //teal600
+    },
```

Comment:
Can we split the colors into primary and secondary groups and label each shade as its number? 
Something like:
`colors: {
primary: {
200: '#99f6e4',
300: '#2dd4bf',
...
},
... }`
Also, it might be useful to have the full range of shades for the teal color too

Reply:
I think `primary` is a special nextui property for the default colour. The other colours can be accessed via css classes from tailwind if wanted (`bg-teal-200` `bg-teal-600` `text-teal-900`) etc.

Reply:
Ahh I see that makes sense. Then it would be better the way it is now